resolwe_bio/processes/workflows/mirna.yml

```diff
@@ -0,0 +1,65 @@
+- slug: workflow-mirna
+  category: workflow:primary
+  name: miRNA pipeline
+  data_name: "miRNA pipeline ({{ reads|sample_name|default('?') }})"
+  requirements:
+    expression-engine: jinja
+  version: 0.0.1
+  type: data:workflow:mirna
+  input:
+      # this set of parameters is for bowtie2
+    - name: reads
+      label: Input miRNA reads.
+      type: data:reads:fastq
+      description: >
+        Note that these reads should already be void of adapters.
+    - name: genome
+      label: Genome
+      type: data:genome:fasta
+      # this set of parameters is for featureCounts
+    - name: annotation
+      label: Annotation (GTF/GFF3)
+      type: data:annotation
+    - name: id_attribute
+      label: ID attribute
+      type: basic:string
+      default: gene_id
+      allow_custom_choice: true
+      choices:
```

Comment:
Fix `id_attribute` input so that labels in choices match the corresponding values. Do the same in `featureCounts` https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/expression/featureCounts.yml#L44.